pyproject.toml

```
@@ -71,6 +71,14 @@ lint.extend-select = [
 lint.isort = {known-first-party = ["helpers", "package_info", "pipx"]}
 lint.mccabe.max-complexity = 15
 
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git,*.pdf,*.svg,.nox'
```

```suggestion
skip = '.git,*.pdf,*.svg,.nox,testdata'
```

Whoops, sorry for the extra trouble.

---

oops, missed this, did myself too. Thanks for the suggestion, and no trouble ;)

---

Let's do it, then.